lib360dataquality/cove/threesixtygiving.py

```diff
@@ -139,6 +140,11 @@ def get_grants_aggregates(json_data):
 
             award_date = str(grant.get("awardDate", ""))
             if award_date:
+                try:
+                    year = award_date[:4]
```

Comment:
What happens if the `award date` is not in a good format?

Reply:
It shouldn't happen I don't think as at that point the data has to be compliant with the standard, if it isn't then as it's acting on a string you'll either get a strange key like "" or another part of the string, this seems acceptable failure mode I think.